http/js/script.js

```diff
@@ -80,19 +80,12 @@ function gaussian(x, a, b, c) {
 }
 
 $(document).ready(function() {
-    $(".footnote a").tipsy({gravity: "w"});
-//    $(".footnote a").on("click", function(e) {
-//        if ($(".tipsy").length === 0) {
-//            $(this).tipsy("show");
-//        }
-//        else {
-//            $(this).tipsy("hide");
-//        }
-//    });
+    // Add tipsy for the footnote.
+    $('.footnote').tipsy({title: 'data-footnote', gravity: 'w', fade: true});
```

Comment:
You've got a point here!

Actually, I was so used to set titles only for `<a>` tags that I forgot that it could be set on every tags. I'll fix that.